app/assets/stylesheets/products.scss

```diff
@@ -3,6 +3,45 @@
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
 .product_image {
+  width: 300px;
+  height: 300px;
+  border-radius: 20%;
+  margin-bottom: 2%;
+}
+
+.product_image_show {
   width: 200px;
   height: 200px;
+  margin-bottom: 10%;
+  margin-left: 5%;
+  border-radius: 20%;
+}
+
+
+.show_page_product {
+  margin-left: 1%;
+  margin-top: 2%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
```

Comment:
You could combine these if you wanted into `flex-flow: row wrap;` but it doesn't make any functional difference